system/api-server/src/file-upload/file-upload.service.ts

```diff
@@ -15,7 +15,7 @@ AWS.config.update({
   secretAccessKey: SECRET_ACCESS_KEY,
 });
 
-@Injectable()
+@Injectable() 
 export class ImageUploadService {
   constructor(
     @InjectRepository(FileUpload)
```

Comment:
Same thing here - why are we injecting a repository? Can you explain what this does?

Reply:
@Injectable() is how you tell the Nest framework that this is a class that can have dependencies that should be instantiated by Nest or something like that